README.md

```diff
@@ -11,6 +11,45 @@ We'd love your help!. Use tags [up-for-grabs][up-for-grabs-issues] and
 
 We have a weekly SIG meeting! See the [community page](https://github.com/open-telemetry/community#javascript-sdk) for meeting details and notes.
 
+## Packages
+
+### API
+
+| Package                 | Description |
+| ----------------------- | -----------------|
+| [Types](https://github.com/open-telemetry/opentelemetry-js/tree/master/packages/opentelemetry-types) | This package provides TypeScript interfaces and enums for the OpenTelemetry core trace and metrics model. It is intended for use both on the server and in the browser. |
+| [Core](https://github.com/open-telemetry/opentelemetry-js/tree/master/packages/opentelemetry-core) | This package provides default and no-op implementations of the OpenTelemetry types for trace and metrics. It's intended for use both on the server and in the browser. |
+
+### Implementation / SDKs
+
+| Package                 | Description |
+| ----------------------- | -----------------|
+| [Basic-Tracer](https://github.com/open-telemetry/opentelemetry-js/tree/master/packages/opentelemetry-basic-tracer) | This module provides a full control over instrumentation and span creation. It doesn't load [`async_hooks`](https://nodejs.org/api/async_hooks.html) or any instrumentation plugin by default. It is intended for use both on the server and in the browser. |
+| [Node-Tracer](https://github.com/open-telemetry/opentelemetry-js/tree/master/packages/opentelemetry-node-tracer) | This module provides automatic tracing for Node.js applications. It is intended for use on the server only. |
+
+### Exporters
+
+OpenTelemetry is vendor-agnostic and can upload data to any backend with various exporter implementations. Even though, OpenTelemetry provides support for many backends, vendors/users can also implement their own exporters for proprietary and unofficially supported backends. Currently, OpenTelemetry supports:
+
+- [Jaeger](https://github.com/open-telemetry/opentelemetry-js/tree/master/packages/opentelemetry-exporter-jaeger)
+- [Zipkin](https://github.com/open-telemetry/opentelemetry-js/tree/master/packages/opentelemetry-exporter-zipkin)
+
+### Plugins
+
+OpenTelemetry can collect tracing data automatically using plugins. Vendors/Users can also create and use their own plugins. Currently, OpenTelemetry supports automatic tracing for:
+
+- [HTTP](https://github.com/open-telemetry/opentelemetry-js/tree/master/packages/opentelemetry-plugin-http)
+- [gRPC](https://github.com/open-telemetry/opentelemetry-js/tree/master/packages/opentelemetry-plugin-grpc)
+
+## Useful links
+- For more information on OpenTelemetry, visit: <https://opentelemetry.io/>
+- For help or feedback on this project, join us on [gitter][node-gitter-url]
```

Comment:
I don't see node-gitter-url ref. I miss something ?

Reply:
This is already present at line no 56.